--- a/src-tauri/src/pairing.rs
+++ b/src-tauri/src/pairing.rs
@@ -38,2 +38,3 @@
     ("SideStore", "ALTPairingFile.mobiledevicepairing"),
+    ("zLoader", "ALTPairingFile.mobiledevicepairing"),
     (
@@ -529,2 +530,2 @@
-        if n == "SideStore" || (live_container && n == "LiveContainer") {
+        if n == "SideStore" || n == "zLoader" || (live_container && n == "LiveContainer") {
             return Ok(Some(PairingAppInfo {
